packages/app/features/profile/screen.tsx

```diff
@@ -1,76 +1,297 @@
 const { useParam } = createParam<{ sendid: string }>()
 interface ProfileScreenProps {
   sendid?: string
 }
 
 export function ProfileScreen({ sendid: propSendid }: ProfileScreenProps) {
-  const { user } = useUser()
   const [paramSendid] = useParam('sendid')
-  const sendid = propSendid || paramSendid
-  const { data: profile, isLoading, error } = useProfileLookup('sendid', sendid || '')
+  const otherUserId = propSendid || paramSendid
+  const { data: otherUserProfile, isLoading, error } = useProfileLookup('sendid', otherUserId || '')
+  const { user, profile: currentUserProfile } = useUser()
   const router = useRouter()
-  const toast = useToastController()
+  const media = useMedia()
 
+  const {
+    data,
+    isLoading: isLoadingActivities,
+    error: activitiesError,
+    isFetching: isFetchingActivities,
+    isFetchingNextPage: isFetchingNextPageActivities,
+    fetchNextPage,
+    hasNextPage,
+  } = useInterUserActivityFeed({
+    pageSize: 10,
+    otherUserId,
+    currentUserId: currentUserProfile?.send_id.toString(),
+  })
+  const { pages } = data ?? {}
   const formatTags = (tags: string[]) => tags?.map((tag) => `/${tag}`).join(' ')
 
   return (
-    <YStack f={1} gap="$6" pt="$6" $gtMd={{ pt: '$0' }}>
-      {error && <Text theme="red">{error.message}</Text>}
-      {isLoading && <Spinner size="large" color="$color10" />}
-      {profile ? (
-        <YStack width="100%" gap="$2">
-          <AvatarProfile profile={profile} /> <H1 nativeID="profileName">{profile.name}</H1>
-          <H2 theme="alt1">{formatTags(profile.all_tags ?? [])}</H2>
-          <Paragraph mb="$4">{profile.about}</Paragraph>
-          {profile && user?.id !== profile?.id ? (
-            <XStack jc="space-around" gap="$6" maxWidth={600}>
-              <Button
-                testID="openSendDialogButton"
-                f={1}
-                width={'100%'}
-                onPress={() => {
-                  router.push({
-                    pathname: '/send',
-                    query: {
-                      recipient: profile.tag ?? profile.sendid,
-                      idType: profile.tag ? 'tag' : 'sendid',
-                    },
-                  })
-                }}
-                theme="green"
-              >
-                Send
-              </Button>
-              <Button
-                f={1}
-                width={'100%'}
-                onPress={() => {
-                  console.log('Request', profile.address)
-                  toast.show('Coming Soon')
-                }}
-              >
-                Request
-              </Button>
-            </XStack>
-          ) : null}
-        </YStack>
-      ) : null}
-    </YStack>
+    <>
+      <YStack f={1} gap="$6">
+        {error && <Text theme="red">{error.message}</Text>}
+        {isLoading && <Spinner size="large" color="$color10" />}
+        {otherUserProfile ? (
+          <YStack h={media.shorter ? '83vh' : '88vh'} $gtMd={{ height: 'auto' }}>
```

Comment:
we will need to start making things compatible with native very soon.

```suggestion
          <YStack h={isWeb ? (media.shorter ? '83vh' : '88vh') : '83%'} $gtMd={{ height: 'auto' }}>
```

Reply:
yes i forgot this wasn't supported on native